### ISSUE TITLE
[Frontend] - Remove page blocks on the Admin Tools Project Configuration tab

### ISSUE THREAD

**Issue by AdityaBoddepalli** [Northeastern-Electric-Racing/FinishLine#2152]
### Description

(Blocked by #2128)
**Important:** Branch off and merge into the feature/admin-tools-redesign branch

The admin tools on the Project configuration tab are wrapped in page blocks. Remove the page blocks for all the tools on that tab. It should look similar to the mocks below when you are done. Keep in mind the mocks aren't all encompassing, they are to give you a feel as to what the page should look like.

### Acceptance Criteria

- [ ] The functionality of the admin tools should remain the same, especially the visibility of tools based on user role.
- [ ] There shouldn't be any page blocks on the project tab.
- [ ] The project configuration tab should include all the tools relating to BOM and the Project Config tool

### Proposed Solution

N/A

### Mocks

![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/68555142/4d777c07-619b-442e-b54c-8d3727954052)
